config/src/test/java/org/springframework/security/config/http/OAuth2ResourceServerBeanDefinitionParserTests.java

```diff
@@ -235,25 +235,26 @@ public void getWhenBearerTokenBeforeNotBeforeThenInvalidToken() throws Exception
 	}
 
 	@Test
-	public void getWhenBearerTokenInTwoPlacesThenInvalidRequest() throws Exception {
-		this.spring.configLocations(xml("JwkSetUri")).autowire();
+	public void getWhenBearerTokenInTwoPlacesAndQueryParameterNotSupportedThenAcceptsRequest() throws Exception {
+		this.spring.configLocations(xml("WebServer"), xml("JwkSetUri")).autowire();
+		mockWebServer(jwks("Default"));
+		String token = this.token("ValidNoScopes");
 		// @formatter:off
-		this.mvc.perform(get("/").header("Authorization", "Bearer token").param("access_token", "token"))
-				.andExpect(status().isBadRequest())
-				.andExpect(invalidRequestHeader("Found multiple bearer tokens in the request"));
+		this.mvc.perform(get("/").header("Authorization", "Bearer " + token).param("access_token", "token"))
+				.andExpect(status().isNotFound());
 		// @formatter:on
 	}
 
 	@Test
-	public void getWhenBearerTokenInTwoParametersThenInvalidRequest() throws Exception {
+	public void getWhenBearerTokenInTwoParametersAndQueryParameterNotSupportedThenUnauthorized() throws Exception {
```

Comment:
Because this is a `GET`, I think we should be able to restore the old test if we add a new check to the `isParameterTokenSupportedForRequest()` method. See below.